Modules/ITS/src/ITSDecodingErrorCheck.cxx

```diff
@@ -31,10 +31,10 @@ namespace o2::quality_control_modules::its
 Quality ITSDecodingErrorCheck::check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap)
 {
   // set timer
-  std::chrono::time_point<std::chrono::high_resolution_clock> start;
-  std::chrono::time_point<std::chrono::high_resolution_clock> end;
-  int TIME;
-  start = std::chrono::high_resolution_clock::now();
+  if (mTFCount == 0) {
```

Comment:
In reality, QCCheck is called each QC Cycle (30 sec to 2 min, depending on task), not TF, so I would rename the variable to `nCycle` to avoid confusion

The situation is different for QC Task, which is indeed called each new TF